### DISCUSSION TITLE
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request

### DISCUSSION THREAD

**Discussion by IvanSeregin** [eclipse-cdo/cdo#7] [Q&A]
Hi all,

I'm trying to build CDO 4.10.0 from source code and caught the below error

```
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}->http://download.eclipse.org:80: The target server failed to respond
INFO: I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {s}->https://download.eclipse.org:443: The target server failed to respond
```

It looks like a bug according to https://bugs.eclipse.org/bugs/show_bug.cgi?id=552876 thread and seems it was fixed; however, I'm still getting this error each time I run mvn install.

The web pages http://download.eclipse.org:80 and  https://download.eclipse.org:443 are accessible from my Google Chrome, so it's not related to my Internet connection or something like that, also I don't use a proxy and tried two different networks.

The maven version I'm currently using is 3.9.4 which is the latest one.

Does anybody know if it's possible to fix the error or is it a remote server related error and I can do nothing with it?

**Comment by estepper** [comment]
Sorry, I have no idea. CDO 4.10 is thirteen releases behind, so chances are that the Eclipse infra has changed too much for an old build description ;-(
